### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by MizouziE**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+For something that is bigger than a one or two line fix:
+1. Create your own fork of the code
+2. Do the changes in your fork
+3. If you like the change and think the project could use it:
+    * Try to follow the code style for the project
+    * Note our Code of Conduct.
+    * Send a pull request.
+
+> **Note**
```
Comment (TAKANOME-DEV) [inline CONTRIBUTING.md:62]: I would suggest that you add the steps from forking to making a PR. 
Here are some examples: 
- https://docs.opensauced.pizza/contributing/introduction-to-contributing/
- https://github.com/EddieHubCommunity/LinkFree/blob/main/CONTRIBUTING.md
- https://github.com/TAKANOME-DEV/coffee-stores/blob/main/CONTRIBUTING.md
- https://docs.nhcarrigan.com/#/contributing

Besides that, everything looks great :+1:

Comment (TAKANOME-DEV) [reply CONTRIBUTING.md:62]: We should also follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for commit messages

Comment (MizouziE) [reply CONTRIBUTING.md:63]: > information_source Suggestion We can follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for commit messages. We can also use [commitlint](https://commitlint.js.org/) and [pre-commit hook](https://prettier.io/docs/en/precommit.html) for linting the commit message.

I'll look into that later cos I think the documentation may need whittling down a bit before adding more steps. Also I am starting to think a bit of a refactor may be on the cards regarding the controllers etc. But all in due time!

Comment (MizouziE) [reply CONTRIBUTING.md:63]: The problems I remember facing are:

1. Ensuring correct version of Node.js
2. Running `git checkout master && git pull` before making a branch (particularly contributors that contribute a **second** time!)
3. Waiting to be assigned an issue

Those topics I think I could maybe cover better. But which parts do I drop?? :sweat_smile:

Comment (TAKANOME-DEV) [reply CONTRIBUTING.md:63]: I think we can force the version of Nodejs to use, have a look [here](https://github.com/blimmer/enforced-package-json-engines)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+- Documentation
+  - Spellchecking and typo corrections are **ALWAYS** welcome :joy:, seriously, I hate typos but make them frequently!
+  - Further detail. If you feel something needs to be expanded on, please do so and submit your PR.
+  - Wiki: If we get to a point that a wiki would be helpful, please open an issue/discussion with your suggestions
+  - As the project grows, this `CONTRIBUTING.md` file will need updating, so add to it where you see fit.
+- Share what you do
+  - In order to encourage others to join in, feel free to share what you have done on your social media/websites
+  - If you know someone that has done something similar, please show them this project as their input would be greatly appreciated
+
+### A link to a resource for people who have never contributed to open source before.
```
Comment (TAKANOME-DEV) [inline CONTRIBUTING.md:46]: Maybe we can drop this resource section for now and if someone struggle to make a contribution, we give it this resource to get started :thinking:

Comment (MizouziE) [reply CONTRIBUTING.md]: done!

File: CONTRIBUTING.md (hunk 1 above)
Comment (MizouziE) [reply CONTRIBUTING.md:58]: I will open a new issue for forcing Node version with a reference to the above

**Pull request merged**